src/Components/App/App.js

```diff
@@ -1,23 +1,30 @@
-import React from 'react';
-import HomeContainer from '../HomeContainer/HomeContainer';
+import React from 'react'
 import { ViewProvider } from '../../Context/ViewContext'
-import './App.css';
-import Header from '../Header/Header';
-import Settings from '../Settings/Settings'
 import { SettingsProvider } from '../../Context/SettingsContext'
+import { Switch, Route } from 'react-router-dom'
+import HomeContainer from '../HomeContainer/HomeContainer'
+import Header from '../Header/Header'
+import Settings from '../Settings/Settings'
+import About from '../About/About'
+import './App.css'
 
 function App() {
   return (
     <section className="App">
       <Header />
-      <SettingsProvider>
-        <ViewProvider>
-          <HomeContainer/>
-        </ViewProvider>
-        <Settings />
-      </SettingsProvider>
+      <Switch>
+        <SettingsProvider>
+          <ViewProvider>
+            <Route exact path="/" render={() => <HomeContainer />} />
+          </ViewProvider>
+          <Route exact path="/settings" render={() => <Settings />} />
+          <Route exact path="/about" render={() => <About />} />
```

Comment:
Loved seeing how this About page is coming together! I'm really glad this is included in the MVP!